YazFi.sh

```diff
@@ -135,24 +164,69 @@ Get_Iface_Var(){
 	echo "$1" | sed -e 's/\.//g'
 }
 
-Get_Guest_Name(){
-	if echo "$1" | grep -q "wl0"; then
-		echo "YazFi 2.4GHz $(echo "$1" | cut -f2 -d".")"
-	elif echo "$1" | grep -q "wl1"; then
-		echo "YazFi 5GHz $(echo "$1" | cut -f2 -d".")"
-	else
-		echo "YazFi 5GHz2 $(echo "$1" | cut -f2 -d".")"
-	fi
+##-------------------------------------##
+## Added by Martinski W. [2022-Dec-23] ##
+##-------------------------------------##
+GetIFaceUILabel()
+{
+	if [ $# -eq 0 ] || [ -z "$1" ] ; then echo "" ; fi
+
+	local theUILabel="*UNKNOWN*"
```

Comment:
This will fail shellcheck as local doesn't exist in POSIX

Reply:
Ah yes, you're right. I've fixed that now and just submitted another PR.

P.S. Although it's marked as "resolved" now, I removed the "local" keyword syntax anyway.